content/spin/v2/language-support-overview.md

```diff
@@ -59,7 +59,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Python"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/spin-python-sdk/)
+**[📄 Visit the Python Spin SDK reference documentation](https://fermyon.github.io/spin-python-sdk/) to see specific modules, functions, variables and syntax relating to the following Python SDK.**
```

Comment:
There's a style shift where some sections relate to "the following <language> features" and some to "the <language> SDK" - is that intentional?